src/vtkqt/vtkQtRenderWindow.C

```diff
@@ -25,7 +25,7 @@
 VTK_MODULE_INIT(vtkRenderingOpenGL2);
 
 
-#ifdef Q_OS_OSX
+#ifdef Q_OS_MAC
```

Comment:
Interestingly enough, Qt has deprecated both of these: Q_OS_OSX and Q_OS_MAC
Looks like Q_OS_MACOS is what should be used.
https://doc.qt.io/qt-5/qtglobal.html#Q_OS_MAC

Reply:
We appear to use `Q_OS_WIN`, Q_OS_LINUX`, `Q_OS_MAC` everywhere except the one instance I changed. Would prefer to keep this one instance consistent with rest of VisIt until all are changed.

Reply:
Understood.